src/natcap/invest/ndr/ndr.py

```diff
@@ -3,10 +3,14 @@
 import logging
 import os
 import pickle
+import shutil
+import tempfile
```

Comment:
Are these used?

Reply:
removed

Reply:
Hmm, looks like they're still in the latest changes ... maybe there's an unpushed commit?

Reply:
Weird, I definitely remember making that change before!